internal/sqlutil/sqlutil.go

```diff
@@ -46,6 +46,9 @@ func Open(dbProperties *config.DatabaseOptions, writer Writer) (*sql.DB, error)
 		db.SetMaxOpenConns(dbProperties.MaxOpenConns())
 		db.SetMaxIdleConns(dbProperties.MaxIdleConns())
 		db.SetConnMaxLifetime(dbProperties.ConnMaxLifetime())
+	} else {
+		db.Exec("PRAGMA busy_timeout = 10000;")
```

Comment:
The linter unhappiness here is because it thinks you might have ignored the return value by accident — I suggest dropping the error explicitly in case it isn't supported on a given underlying platform (like WASM), so in that case you need to do `_ = db.Exec(...`.